test/browser-test/generic/index.js

```diff
@@ -0,0 +1,34 @@
+const tf = require("@tensorflow/tfjs-core");
+
+describe("Generic Split test in browser", () => {
+  it("should split data into train & test", async () => {
+ 
+    const X = tf.range(0, 10).reshape([5, 2]);
+
+    const y = tf.range(0, 5);
+
+    const [X_train, X_test, y_train, y_test] = datacook.Generic.split([X, y]);
```

Comment:
Why does the X to be uppercase?

Reply:
this aligns to python convention

Reply:
Hmm, why do we keep python convention on a JS project?